src/Connectors/test/Connector.EF6Core.Test/MySqlTypeLocatorTest.cs

```diff
@@ -44,4 +37,3 @@ public void Driver_Found_In_MySqlData_Assembly()
         }
```

Comment:
Which line? The github UI might have done something confusing here with collapsing sections

Reply:
Line 36: 
```
     // assert
            Assert.NotNull(type);

this one -> MySqlTypeLocator.Assemblies = types;


     }
```

Reply:
This line resets the array back to the default list of assemblies to search for MySQL types. My recollection is that without it, other tests could fail, though that seems odd since all tests in this assembly should work with whichever one passes here... If it isn't needed anymore, there are several other places that it could be removed from, but it hasn't changed recently